src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -97,6 +99,18 @@ private static void setupLogger(Application application) {
 		application.addComponentMenuElement(Logger.class, "OFF logging", (ActionEvent e) -> logger.setLevel(Level.OFF));
 	}
 
+	private static void setMouseDrawer(Application application) {
+		application.getFreePanel().addMouseListener(new MouseInputAdapter() {
+			@Override
+			public void mouseClicked(java.awt.event.MouseEvent e) {
+				int correctedX = e.getX() - 268;
+				int correctedY = e.getY() - 227;
```

Comment:
How did you get those values? 
They are not arbitrary constants so they should be calculated